src/molecules/Tabs.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { NavLink, Switch, Route } from 'react-router-dom'
+import styled from 'styled-components'
+import { Box } from 'atomic-layout'
+
+const StyledLink = styled(NavLink)`
+  color: ${({ theme }) => theme.colors.greyLight};
+  padding: 0.5rem 0;
+  font-size: 22px;
+  font-weight: 600;
+
+  ::after {
+    content: '';
+    display: block;
+    width: 0;
+    height: 2.5px;
+    background-color: ${({ theme }) => theme.colors.orange};
+    transition: width ease 0.35s;
+  }
+
+  &.active {
+    color: ${({ theme }) => theme.colors.blackLight};
+  }
+
+  :hover,
+  &.active {
+    ::after {
+      width: 100%;
+    }
+  }
+`
+
+const StyldUl = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin-bottom: 55px;
+  text-align: center;
+`
+
+const StyledLI = styled.li`
+  display: inline-block;
+
+  :not(:last-child) {
+    margin-right: 45px;
+  }
+`
+
+const Tabs: React.FC = () => {
+  return (
+    <Box marginBottom="42px">
+      <StyldUl>
+        <StyledLI>
+          <StyledLink exact to="/product/">
+            Description
+          </StyledLink>
+        </StyledLI>
+        <StyledLI>
+          <StyledLink to="/product/info">Info</StyledLink>
+        </StyledLI>
+        <StyledLI>
+          <StyledLink to="/product/review">Review</StyledLink>
+        </StyledLI>
+      </StyldUl>
+      <Switch>
+        <Route
+          path="/product/"
+          exact
+          render={() => (
+            <p>
+              I'm description Lorem ipsum dolor sit amet consectetur adipisicing
+              elit. Quo sapiente tenetur cum, quis, quia veniam quod fugit
+              eligendi cupiditate vero, sequi asperiores cumque eos ipsum
+              pariatur alias dolore saepe unde.
+            </p>
+          )}
+        />
+        <Route
+          path="/product/info"
```

Comment:
It would be great to reuse the parent's route `path` here, not to repeat it.

Previously, I used to do it with:

```jsx
import { withRouter, Route } from 'react-router-dom'

// Wrapping "MyComponent" with "withRouter()" give it the "match" prop automagically.
const MyComponent = ({ match }) => (
  return <Route path={`${match.url}/info`} />
)

export { MyComponent: withRouter(MyComponent) }
```

Right now I see there are some React hooks in `react-router`, so maybe try to use one of them. For example, [`useRouteMatch`](https://reacttraining.com/react-router/web/api/Hooks/useroutematch). I can't guarantee this would work, but you can always try what I've suggested above, if it doesn't.